bot/src/discord/listeners/ReadyListener.ts

```diff
@@ -1,14 +1,11 @@
-import {DiscordListener} from "../DiscordListener";
-import {AirhornBot} from "../../bot";
+import { Client } from 'discord.js';
+import { log } from '../../bot';
+import { registerCommandsOnDiscord } from './InteractionCreateListener';
 
-export class ReadyListener extends DiscordListener {
-
-  registerListener(airhornBot: AirhornBot): void {
-    airhornBot.client.on("ready", async () => {
-      console.log("AIRHORN SOLUTIONS is ready.");
-    });
-    airhornBot.client.on("shardReady", async (shardId: number) => {
-      console.log("Shard ID " + shardId + " is ready.");
-    });
+export async function readyListener(clientObject: Client<true>): Promise<void> {
+  log.info(`Logged in as ${clientObject.user.tag} (${clientObject.user.id}).`);
+  if (process.env.REGISTER_DISCORD_COMMANDS === 'true') {
```

Comment:
We can save future us a lot of pain by hitting this with a `.toLowerCase()` if it's defined